schemas/oic.wk.res-schema.json

```diff
@@ -8,24 +8,20 @@
       "properties": {
```

Comment:
Wasn't /oic/res also changing to just be a flat array and not require di?  I know this is the baseline schema but the ll schema needs cleaning up (I think) otherwise we don't align with what is in the Spec Section 11.3.5